static/metricsgraphics/css/bootstrap.min.css

```diff
@@ -0,0 +1,5 @@
+/*!
+ * Bootstrap v3.2.0 (http://getbootstrap.com)
```

Comment:
Bootstrap 3.2.0 was released on June 26th, 2014.

If this were a Perl module, I'd be looking for a patch above that adds a dependency to the build process. Does the build process currently understand vendored static web content like Bootstrap and font-awesome?

Reply:
I'd need to pull in a new version and test it to see if it works. I'm simply using the version here that I've always been using since I coded the original Bugzilla Socorro Lens extension.